### DISCUSSION TITLE
Passing composed trees to another composer

### DISCUSSION THREAD

**Discussion by thumannw** [cashapp/molecule#107] [Q&A] [answered]
Your readme shows an example of how to pass a composed object by `molecule` to Jetpack Compose UI. It is not yet clear to me how this should look like when the object is a full-blown tree instead. Could you show such an example? I guess the goal is that the downstream composer (Jetpack Compose UI) only recomposes the parts which were also recomposed by the upstream composer (molecule).

**Comment by JakeWharton** [comment, marked as answer]
Molecule does not use Compose's tree management facilities. The only mechanism by which you can pass things out of Molecule is as a single value.
